common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -380,22 +380,21 @@ function(VideoPlayer, HLS) {
             });
 
             describe('when the video is playing', function() {
-                beforeEach(function(done) {
+                beforeEach(function() {
                     state.videoPlayer.play();
+                });
 
+                it('call runTimer in seekTo on player', function(done) {
                     jasmine.waitUntil(function() {
```

Comment:
@muhammad-ammar I am not getting why do we need `waitUntil` only in this test, may you please provide some context behind?